src/main/scala/com/atomist/rug/function/github/InstallOrgWebHookFunction.scala

```diff
@@ -27,10 +25,8 @@ class InstallOrgWebHookFunction extends AnnotatedRugFunction
 
     try {
       val ghs = gitHubServices(token, apiUrl)
-      val org = ghs.gitHub.getOrganization(owner)
-      val config = Map("url" -> url, "content_type" -> "json")
-      val gHHook = org.createHook("web", config.asJava, Seq(GHEvent.ALL).asJava, true)
-      val response = mapHook(gHHook)
+      val wh = ghs.createOrganizationWebhook(owner, "web", url, "json", active = true, Events.toArray)
```

Comment:
Here you use the dot notation, which I prefer, while others you use a space. Should we try to be consistent?